docs/server/source/events/websocket-event-stream-api.rst

```diff
@@ -70,22 +70,22 @@ All messages sent in a stream are in the JSON format.
 .. note::
 
     For simplicity, BigchainDB initially only provides a stream for all
-    validated transactions. In the future, we may provide streams for other
+    committed transactions. In the future, we may provide streams for other
     information. We may
     also provide the ability to filter the stream for specific qualities, such
     as a specific ``output``'s ``public_key``.
 
     If you have specific use cases that you think would fit as part of this
-    API, feel free to reach out via `Gitter <https://gitter.im/bigchaindb/bigchaindb>`_
-    or `email <mailto:product@bigchaindb.com>`_.
+    API, consider creating a new `BEP <https://github.com/bigchaindb/BEPs>`_.
 
 Valid Transactions
 ~~~~~~~~~~~~~~~~~~
 
 ``/valid_transactions``
 
-Streams an event for any newly validated transactions. Message bodies contain
-the transaction's ID, associated asset ID, and containing block's ID.
+Streams an event for any newly valid transactions committed to a block. Message
+bodies contain the transaction's ID, associated asset ID, and containing
+block's ID.
```

Comment:
@kansi @ttmc when subscribing to events, the `block_id` is put into a `string`. I wonder if we should change it to `int` since it's the *height* of the block.

Reply:
@vrde I agree, it should be an int

Reply:
Thanks @kansi. I can address this in another PR once we merge this.

Reply:
I created a new issue #2166 so we don't forget to do that.